lib/Metrc.js

```diff
@@ -186,10 +202,13 @@ function Metrc(domain, licenseNumber, apiKey, userKey) {
   }
   
   return {
-    get: get,
+    get,
+    post,
+    put,
     post: post,
```

Comment:
error?

Reply:
ah good catch, i just committed from the comments ☝️without checking if it was proper; will fix, thanks!